packages/gamut/src/AppBar/index.tsx

```diff
@@ -21,7 +21,10 @@ export const AppBar: React.FC<AppBarProps> = ({
 
   return (
     <div className={classes}>
-      <ContentContainer className={styles.contentWrapper} wide={wide}>
+      <ContentContainer
+        className={styles.contentWrapper}
+        size={wide ? 'wide' : 'medium'}
+      >
         {children}
       </ContentContainer>
```

Comment:
We can remove that classname altogether by restructuring the html a bit :)

```suggestion
      <ContentContainer  size={wide ? 'wide' : 'medium'}>
       <FlexBox alignItems="center">{children}</FlexBox>
    </ContentContainer>
```